spring-boot-samples/spring-boot-sample-data-geode/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<!-- Your own application should inherit from spring-boot-starter-parent -->
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-samples</artifactId>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-boot-sample-data-geode</artifactId>
+	<name>Spring Boot Data Geode Sample</name>
+	<description>Spring Boot Data Geode Sample</description>
+	<url>http://projects.spring.io/spring-boot/</url>
+	<organization>
+		<name>Pivotal Software, Inc.</name>
+		<url>http://www.spring.io</url>
+	</organization>
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+		<spring-shell.version>1.1.0.RELEASE</spring-shell.version>
```

Comment:
All dependencies in Spring Boot have to be declared in `spring-boot-dependencies`. A sample should be as straightforward as possible: if you have to define special entries or exclusion in the sample it means that everyone else should. This defeats the purpose of the starter.

If you want to showcase the "Manager" case in the sample, that's perfectly fine to add `spring-shell` explicitly (i.e. the starter should not bring _everything_ that is possible). But it shouldn't have a version there.